web_app/calendar/app.py

```diff
@@ -0,0 +1,165 @@
+import json
+from copy import copy
+
+from flask import Flask
+from html_template import *
+
+app = Flask(__name__)
+
+
+with open('date_template.json', 'r') as file:
+    date_template = json.loads(file.read())
+
+
+@app.route(years_route)
+def get_years_page():
+    # ToDo Get years from date_template and form page
+    return body_html.format(year_table)
+
+
+@app.route(months_route)
+def get_months_page_for_year(year_id):
+    # ToDo(den) add logic
+    # https://github.com/stasya72008/highway-to-hell/issues/13
+    # remove -----------
+    month_busy.update({'May': "#eee", 'May_task_count': " (3)"})
+    month_busy.update({'April': "#eee", 'April_task_count': " (1)"})
+    # ------------------
+
+    month = month_table.format(year_id=year_id,
+                               **month_busy)
+    return body_html.format(month)
+
+
+@app.route(days_route)
+def get_days_page_for_month(year_id, month_id):
+    # ToDo(den) add logic
+    # https://github.com/stasya72008/highway-to-hell/issues/13
+    # tasks = get_task_for_rest()
+
+    # ToDo add switch to previous and next year
+    pr_month_id = 1 if month_id == 1 else month_id - 1
```

Comment:
Please, rename the variables to prev_month_id and next_month_id for the sake of readability :)